irods_specific_implementation/utils.hpp

```diff
@@ -63,6 +64,12 @@ namespace irods::rest
 
         _response.send(http_code, msg);
     } // handle_request
+
+    inline bool to_bool(std::string& s) noexcept {
+        boost::algorithm::to_lower(s);
+        return s == "1" || s == "true";
```

Comment:
Sorry, I forgot what we decided on this. Just leaving as-is? I seem to recall only supporting "1"...